Adyen/Core/Components/StoredPaymentMethodComponent.swift

```diff
@@ -81,19 +81,14 @@ extension StoredPaymentMethodComponent {
     /// Configuration for Stored Payment type components.
     public struct Configuration: AnyBasicComponentConfiguration {
 
-        /// A Boolean value that determines whether the payment button is displayed. Defaults to `true`.
-        public private(set) var showSubmitButton: Bool
-
         public var localizationParameters: LocalizationParameters?
 
         /// Initializes the configuration for Issuer list type components.
         /// - Parameters:
-        ///   - showSubmitButton: Boolean value that determines whether the payment button is displayed.
+        ///   - showsSubmitButton: Boolean value that determines whether the payment button is displayed.
```

Comment:
The function does not have a `showsSubmitButton` parameter anymore

Reply:
Good catch. Removed